adafruit_lsm6ds.py

```diff
@@ -386,6 +405,33 @@ def __init__(self, i2c_bus, address=_LSM6DS_DEFAULT_ADDRESS):
         self._i3c_disable = True
 
 
+class LSM6DSO32(LSM6DS):  # pylint: disable=too-many-instance-attributes
```

Comment:
How about splitting the subclasses into separate files? That way they all aren't loaded into memory when one is needed.

Reply:
@tannewt ok, please take a look